src/operators/validate_dtd.h

```diff
@@ -33,18 +33,30 @@
 namespace modsecurity {
 namespace operators {
 
-class ValidateDTD : public Operator {
+class XmlDtdPtrManager {
  public:
-    /** @ingroup ModSecurity_Operator */
```

Comment:
This marking is further used by doxygen, in order to place this class under the operators sub class.